p2p/src/network.rs

```diff
@@ -191,40 +192,40 @@ struct NetworkBase<T: Pload, K: Kex, E: Enc> {
 
 impl<T: Pload, K: Kex, E: Enc> NetworkBase<T, K, E> {
     /// [`Self`] task.
+    #[log(skip(self), fields(listen_addr=self.listen_addr, public_key=%self.public_key))]
     async fn run(mut self) {
         // TODO: probably should be configuration parameter
-        let mut update_topology_interval = tokio::time::interval(Duration::from_millis(1000));
+        let mut update_topology_interval = tokio::time::interval(Duration::from_millis(100));
```

Comment:
Ideally we shouldn't have these long-term. However, I'm conscious that sometimes arbitrary numbers have to be pulled out of a hat to make things work.

Reply:
I agree, in theory it should be possible to remove periodic topology update at all.
We will just try to reconnect if previous connection was terminated.
But for know i think periodic approach is ok.